models/UserSwitch.php

```diff
@@ -61,7 +61,7 @@ public function attributeLabels()
     public function getUser()
     {
         if (empty($this->user)) {
-            $this->user = \Yii::$app->user;
+            $this->user = Yii::$app->get('user');
```

Comment:
Any of these options does the same thing, but the last will work a little faster than all the previous.
`Yii::$app->user;` < `Yii::$app->getUser();` < `Yii::$app->get('user');`